okhttp/src/main/java/com/squareup/okhttp/Call.java

```diff
@@ -270,6 +270,10 @@ Response getResponse(Request request, boolean forWebSocket) throws IOException {
       try {
         engine.sendRequest();
         engine.readResponse();
+      } catch (ConnectionFailureException e) {
+        // TODO(nfuller): It would be nice to log or expose all exceptions somehow. E.g. >= JDK1.7
+        // with "suppressed" exceptions.
```

Comment:
nit: scare quotes aren't necessary; you can just say "with suppressed exceptions."